twilio/twilio.go

```diff
@@ -97,9 +97,9 @@ type Meta struct {
 	URL             *string `json:"url"`
 }
 
-// NewClient provides an initialized Twilio client.
-func NewClient(accountSID string, authToken string) *Twilio {
-	credentials := &client.Credentials{AccountSID: accountSID, AuthToken: authToken}
+// NewClientWithAccountSid provides an initialized Twilio client with accountSid.
+func NewClientWithAccountSid(username string, password string, accountSid string) *Twilio {
```

Comment:
I've a feeling we'll be wanting more optional params when constructing a `Twilio`. Can this be changed to accept a `struct` of optional params?

Reply:
Optional params such as edge and region? Any ideas on what to call this func?